core/src/main/java/hivemall/fm/FieldAwareFactorizationMachineModel.java

```diff
@@ -259,9 +255,9 @@ protected final float eta(@Nonnull final Entry theta, final long t, final float
     protected final float eta(@Nonnull final Entry theta, @Nonnegative final int f, final long t,
             final float grad) {
         if (_useAdaGrad) {
-            double gg = theta.getSumOfSquaredGradients(f);
```

Comment:
@takuti This behavior (that used in libffm) is wrong in strict sense and previous code is much better because initial eta should equals to `eta0` but this implementation depends on the initial gradient.